tests/clients/test_aws_cost_usage_client.py

```diff
@@ -0,0 +1,34 @@
+from datetime import date
+from tests.aws_scanner_test_case import AwsScannerTestCase
+from unittest.mock import Mock
+
+from src.clients.aws_cost_usage_client import AwsCostUsageClient, CostUsageException
+from botocore.exceptions import BotoCoreError
+from tests.clients.test_aws_cost_usage_responses import GET_USAGE_COST_SUCCESS
+
+
+class TestAwsCostUsageClient(AwsScannerTestCase):
+    def test_get_aws_cost_usage_success(self) -> None:
+        boto_cost_usage = Mock(get_cost_and_usage=Mock(return_value=GET_USAGE_COST_SUCCESS))
```

Comment:
There's no argument matching on this mock. An assertion is needed to ensure that correct arguments are passed to the `get_cost_and_usage` function of the client.

Reply:
Does def test_parse_cli_args_for_cost_usage_task() in test_aws_scanner_argument_parser not take care of this?